stable/minio/templates/deployment.yaml

```diff
@@ -147,4 +147,16 @@ spec:
             secretName: {{ if .Values.existingSecret }}{{ .Values.existingSecret }}{{ else }}{{ template "minio.fullname" . }}{{ end }}
         - name: minio-config-dir
           emptyDir: {}
+        {{- if .Values.tls.enabled }}
+        - name: cert-secret-volume
+          secret:
+            secretName: {{ .Values.tls.certSecret }}
+            items:
+            - key: public.crt
```

Comment:
It would be nice if we can define use something like `{{  .Values.tls.publicKey }}` and `{{  .Values.tls.privateKey }}`.
For example, if you use `cert-manager` to issue the certificates you'll end with a secret with the keys `tls.crt` and `tls.key`.

Reply:
Sure will do that @oeN